### ISSUE TITLE
scp is missing

### ISSUE THREAD

**Issue by hfavisado** [cizia64/CrossMix-OS#240]
# Attempt to copy files from host machine to TSP using scp:
i.e., in host machine:
`scp <file> <root@ip>:/mnt/SDCARD/RetroArch/.retroarch/saves/PS/PCSX-ReARMed/`

Error:
```
ash: scp: not found
lost connection
```

# Attempt to copy files from TSP to a different machine using scp
i.e., in TSP:
`scp <file> <target-user@target-ip>:<target-path>`

Error:
```
-ash: scp: not found
```

**Comment by Nevrdid** [comment]
I just use sshd on my linux device to move thing on both direction.
For host to tsp, it work for me without change with `scp <src> root@192.168.33.214:/mnt/SDCARD/<dest>`
We can add scp to tsp but imo sshd is a way easier solution.

**Comment by hfavisado** [comment]
Sorry, I didnt look into it deep enough. I actually wanted to use rsync and thought initially that rsync used scp internally for secure connections, but after reading some more, I found out that that was false. I will try to edit this issue to mention rsync instead.

**Comment by Nevrdid** [comment]
Yeah, the issue beeing that the PATH missed /mnt/SDCARD/System/bin when you login in a new tty. You can fix it adding `export PATH=/mnt/SDCARD/System/bin:$PATH` to `/etc/profile`.

**Comment by hfavisado** [comment]
Thanks. Editing `PATH` in `/etc/profile` works only if logging in via tty, meaning, if I ssh into the TSP and run rsync from there. If I use rsync on a different device to the TSP, i.e. `rsync <folder> root@TSP-IP-addr:/path/`, the error `rsync not found` is returned. I tried editing `/etc/preinit` because I was able to grep a line setting the `PATH` variable there, but that didn't work either. How can I enable rsync to work without logging into tty?

**Comment by hfavisado** [comment]
Note: The following works (adding ` --rsync-path=/mnt/SDCARD/System/bin/rsync`) but what I'm trying to look for is if there's a way to not have to pass that parameter.

```
rsync root@192.168.0.123:/mnt/SDCARD/BIOS/ ~/Desktop/ -avn --progress --rsync-path=/mnt/SDCARD/System/bin/rsync
```

**Comment by Nevrdid** [comment]
`cp /mnt/SDCARD/System/rsync /user/bin`

**Comment by hfavisado** [comment]
I see. Thanks for the workaround. I hope somehow we can add `/mnt/SDCARD/System/bin` to the default path without logging into tty. I couldn't figure out where it's set. (tried `/etc/profile` and `/etc/preinit` but neither worked)
